src/anm/graphics/text.js

```diff
@@ -171,7 +171,7 @@ Text.prototype.bounds = function() {
  * @param {Number} pt.y
  * @return {Boolean} is point inside
  */
-Text.prototype.inside = function(pt) {
+Text.prototype.contains = function(pt) {
```

Comment:
well, this optimization implies that "contains" always go after "inBounds". is it really true?